### ISSUE TITLE
UI detach

### ISSUE THREAD

**Issue by davidsowerby** [KrailOrg/krail#26]
As part of #24 I tried to use ScopeUI.detach() to release the UI from the UIScope cache, am unable to test it.  The detach() method does not appear to be called - or it may just be that the framework deals with it in as part of its own caching, given that the close() method is just a signal for the framework to close the UI.  A question for the Vaadin team.
